dabplus-on-air-processing.spec

```diff
@@ -11,7 +11,7 @@ Name:     dabplus-on-air-processing
 BuildRequires: make
 BuildRequires: systemd
 
-Requires: liquidsoap
+Requires: liquidsoap = ${_ls_version}
```

Comment:
I didn't figure out a smart way to use the `%include "file"` directive without having to hardcode `"file"` in some way or other (ie. for `/usr/local` installs). Having it this way lets me try the code against an opam install of liquidsoap by manually deploying it to the opam libdir.

Maybe I could do something relative based on the libdir with `%include <../%{name}/file>`. Liquidsoap 1.3.1 is coming soon. If that's ok with you, I'll give relative `<>` includes a try when I update to it (I'll have a real test rig set up then).